settings.json.sample

```diff
@@ -1,9 +1,9 @@
 {
-    "API_URL": "localhost:5417", # The API URL
-    "SECURE_HTTP": false, # Is the API protected by HTTPS?
-    "templates": { # Templates sample
-        "basic": { # Template name
-            "description": "Basic template", # Template description
+    "API_URL": "http://url.to.salt.api:5417",
```

Comment:
Please don't include the http protocol, it's override by the SECURE_HTTP config key. I'm not sure if we should change the API_URL value.

For the comments, they are here for pushing people to customize the settings file, I've made recent changes (not yet pushed) to better report errors on the interface when comments were not removed.